_dprhtml/js/analysis_output.js

```diff
@@ -300,6 +300,6 @@ async function reanalyze(word,cancel) {
 return {
 outputDef : outputDef,
 reanalyze : reanalyze,
-reanalyzeOnEnterKey : reanalyzeOnEnterKey,
+reanalyzeOnEnterKey : reanalyzeOnEnterKey
```

Comment:
good practice to retain the trailing ',' so next time you add another line below, just 1 diff will show up.